### PR TITLE
Add "Know the community page"

### PR DESCRIPTION
**Pull request by ORKO06**
# Pull Request Template

## Description

I have added the know the community page. I made a custom card slider for the page

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue).
- [x] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] This change requires a documentation update.

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

## Additional Context (Please include any Screenshots/gifs if relevant)

...

## Checklist:

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have made corresponding changes to the documentation.
- [ ] I have added corresponding tests.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I have signed the commit message to agree to Developer Certificate of Origin (DCO) (to certify that you wrote or otherwise have the right to submit your contribution to the project.) by adding "--signoff" to my git commit command.

### CODE REVIEW

File: website/src/components/KnowTheCommunityCarousel/index.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import styles from './index.module.css';
+import features from '../../../websiteData/features';
+import { PhCaretLeftFill, PhCaretRightFill } from '../../assets/icon';
+
+const index = () => {
+  const [curSlide, setCurSlide] = useState(0);
+
+  const onLeftClick = () => {
+    if (curSlide === 0) {
+      setCurSlide(-8);
```
Comment (padmajabhol) [inline website/src/components/KnowTheCommunityCarousel/index.js:12]: Hardcoding this value leaves extra space at the end. Can you brainstorm on how you can make this dynamic?

File: website/src/components/KnowTheCommunityCarousel/index.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react';
+import KnowTheCommunityCard from '../KnowTheCommunityCard/index';
+import styles from './index.module.css';
+import features from '../../../websiteData/features';
+import { PhCaretLeftFill, PhCaretRightFill } from '../../assets/icon';
+
+const index = () => {
```
Comment (padmajabhol) [inline website/src/components/KnowTheCommunityCarousel/index.js:7]: Please change "index" to a proper component name

**Review by padmajabhol — CHANGES_REQUESTED** [review]
We can further on to make the carousel responsive.

**Comment by padmajabhol** [comment]
@sohamsshah can you please look into the carousel logic and maybe ?

File: website/src/components/KnowTheCommunityCarousel/index.js (hunk 1 above)
Comment (ORKO06) [reply website/src/components/KnowTheCommunityCarousel/index.js:12]: We can make that dynamic by changing that hard-coded value depending on the screen size

File: website/src/components/KnowTheCommunityCarousel/index.js (hunk 2 above)
Comment (ORKO06) [reply website/src/components/KnowTheCommunityCarousel/index.js:7]: Sure I will change it. Thanks for commenting that out

File: website/src/components/KnowTheCommunityCarousel/index.js (hunk 1 above)
Comment (padmajabhol) [reply website/src/components/KnowTheCommunityCarousel/index.js:12]: Or maybe you can set the number of clicks depending on how many screens are being rendered. Either works.

**Review by padmajabhol — CHANGES_REQUESTED** [review]
The misc. terminal box is hidden for medium and smaller screens, you can change it's positioning to not over lap with the hero section.
After the above issues are fixed, we can proceed to make it mobile friendly.

**Comment by padmajabhol** [comment]
Hi @ORKO06 , are you still working on this? If not, we can assign it to someone else. Please respond.

**Pull request closed without merging**